astropy/visualization/wcsaxes/core.py

```diff
@@ -705,7 +705,7 @@ def get_transform(self, frame):
         """
         Return a transform from the specified frame to display coordinates.
 
-        This does not include the transData transformation
+        This includes the transData transformation
```

Comment:
Huh. Looks like @astrofrog wrote this text back in #5496 when `wcsaxes` was transferred into `astropy`. Maybe he can clarify.